index.html

```diff
@@ -0,0 +1,3 @@
+<h1>
+    Hello Microverse!
+</h1>
```

Comment:
- [ ] [REQUIRED] Good job so far! A `.html` file that includes a `header` with the text "Hello Microverse!". A .css file that includes one class with styles for your header (make it in your favorite color). But it can not working on your side because you forgot to implement [`HTML5 => DOCTYPE html`](https://www.w3docs.com/snippets/html/html5-page-structure.html) and include the `CSS` file. Please fix it 🙏🏼